dome9CloudBots/bots/modify_network_security_group_scope_by_port.py

```diff
@@ -0,0 +1,69 @@
+# What it does: Change network security group scope by a given port.
+# Scope can be list of ip addresses with , between. example: 192.168.99.0/24,10.0.0.0/24,44.66.0.0/24
+# Direction can be : source or destination
+# Bot will remove Any from direction
+# Usage: AUTO: modify_network_security_group_scope_by_port <port> <direction> <scope>
+# Example: AUTO: modify_network_security_group_scope_by_port 556 source 10.0.0.0/24,172.16.0.1/32,168.243.22.0/23
+# Limitations: None
+
+from msrestazure.azure_exceptions import CloudError
+from azure.mgmt.network import NetworkManagementClient
+import logging
+
+
+def modify_scope(rule, direction, scope):
+    if len(scope) == 1:
+        rule[f'{direction}_address_prefix'] = scope[0]
+        rule[f'{direction}_address_prefixes'] = None
+    else:
+        rule[f'{direction}_address_prefixes'] = scope
+        rule[f'{direction}_address_prefix'] = None
```

Comment:
rule[f'{direction}_address_prefix'] = None happen on both cases 
why not use it once ?

Reply:
There is :
'{direction}_address_prefixes'
'{direction}_address_prefix'